tests/utils.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals, absolute_import
+
+from datetime import datetime
+
+from estuary.models.koji import KojiBuild
+import koji
+import pytz
+
+
+def mock_getBuild(state=koji.BUILD_STATES['COMPLETE'], isCommit=False):
```

Comment:
`isCommit` => `is_commit`

Reply:
It seems like what you're trying to get at is if this is a container or not with your `isCommit` flag. I suggest something along the lines of `is_container` instead.